conf/vsc_ugent.config

```diff
@@ -0,0 +1,115 @@
+// Specify the work directory
+workDir = "$VSC_SCRATCH_VO_USER/work"
+
+// Perform work directory cleanup when the run has succesfully completed
+cleanup = true
+
+// Reduce the job submit rate to about 10 per second, this way the server won't be bombarded with jobs
+executor {
+    submitRateLimit = '10 sec'
+}
+
+// Specify that singularity should be used and where the cache dir will be for the images
+singularity {
+    enabled = true
+    autoMounts = true
+    cacheDir = "$VSC_SCRATCH_VO_USER/singularity"
+}
+
+// Define profiles for each cluster
+profiles {
+    skitty {
+        params {
+            config_profile_description = 'HPC_SKITTY profile for use on the Skitty cluster of the VSC HPC.'
+            config_profile_contact = 'Nicolas Vannieuwkerke (@nvnieuwk)'
+            config_profile_url = 'https://www.ugent.be/hpc/en'
+            max_memory = 177.GB
+            max_cpus = 36
+            max_time = 72.h
+        }
+
+        process {
+            executor = 'slurm'
+            queue = 'skitty'
+            maxRetries = 2
+            beforeScript = "export SINGULARITY_CACHEDIR=$VSC_SCRATCH_VO_USER/.singularity"
+            scratch = "$VSC_SCRATCH_VO_USER"
+        }
+    }
+
+    swalot {
+        params {
+            config_profile_description = 'HPC_SWALOT profile for use on the Swalot cluster of the VSC HPC.'
+            config_profile_contact = 'Nicolas Vannieuwkerke (@nvnieuwk)'
+            config_profile_url = 'https://www.ugent.be/hpc/en'
+            max_memory = 116.GB
+            max_cpus = 20
+            max_time = 72.h
```

Comment:
You can probably cut this down a bit by adding the common stuff up at the top under the main `params` scope. `config_profile_contact`, `config_profile_url` and `max_time` seem to be the same in most / all profiles, so you could define them there once to be applied under the `-profile vsc_ugent` option and then inherit / overwrite them in the subprofiles (`-profile vsc_ugent,swalot`). Make sense?

Reply:
The nf-core tutorial says it's not a good idea to do this (https://nf-co.re/docs/usage/tutorials/step_by_step_institutional_profile#profiles-scope), or maybe I misunderstood the warning about it :p

Reply:
You're totally right! My apologies. I had completely forgotten this.

Reply:
No problem! Thanks for all the feedback!

Reply:
I'm sure that this _used_ to work, possibly many years ago 🤔 

https://github.com/nf-core/configs/blob/1f872aea87757635bc705e3340ee8e5c2998be77/conf/uppmax.config#L111-L124

But yeah stay on the safe side for sure 👍🏻